front/lib/create_complaint/complaint_page.dart

```diff
@@ -0,0 +1,158 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:mc426_front/create_complaint/complaint.dart';
+import 'package:mc426_front/create_complaint/ui/bloc/complaint_bloc.dart';
+
+class ComplaintPage extends StatefulWidget {
+  static const String routeName = '/complaint';
+
+  const ComplaintPage({super.key});
+
+  @override
+  State<ComplaintPage> createState() => _ComplaintPageState();
+}
+
+class _ComplaintPageState extends State<ComplaintPage> {
+
+TextEditingController titleController = TextEditingController();
+  TextEditingController descriptionController = TextEditingController();
+  TextEditingController addressController = TextEditingController();
+  bool isAnonymous = false;
+  
+  final _bloc = ComplaintBloc();
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocConsumer<ComplaintBloc, ComplaintState>(
+      bloc: _bloc,
+      listener: (context, state) {
+        if (state is ComplaintLoadedState && state.result != null) {
+          ScaffoldMessenger.of(context).showSnackBar(
+            SnackBar(
+              duration: const Duration(milliseconds: 1500),
+              content: Text(state.result!.message),
+              backgroundColor: state.result!.isSuccess ? Colors.green : Colors.red,
+            ),
+          );
+        }
+      },
+      builder: (context, state) {
+        return Scaffold(
+          appBar: AppBar(
+            centerTitle: true,
+            title: const Text(
+              'Criar Denúncia',
+              style: TextStyle(
+              fontSize: 26.0,
+              fontWeight: FontWeight.bold,
+              color: Colors.white,
+              ),
+            ),
+            backgroundColor: const Color(0xFF4CE5B1),
+          ),
+          body: Padding(
+            padding: const EdgeInsets.only(top: 150.0, right: 16.0, left: 16.0),
```

Comment:
Não é bom colocar um padding por cima de um SingleChildScrollView, se for pra ter, melhor que seja dentro, aqui eu recomendo ao invés do padding top colocar um SizedBox com o tamanho desejado dentro do Column e manter apenas os paddings laterais. Aumentaria os paddings laterais para 24 tbm e 150 precisaria ou ser 152 ou 148

Reply:
Algo assim 
`body: SingleChildScrollView(  // Adicionado SingleChildScrollView
            child: Padding(
              padding: const EdgeInsets.symmetric(horizontal: 24),
              child: Column(
                mainAxisAlignment: MainAxisAlignment.center,
                crossAxisAlignment: CrossAxisAlignment.stretch,
                children: [
                  const SizedBox(height: 148),
                  TextFormField(`